canvas_sdk/v1/data/base.py

```diff
@@ -158,7 +165,13 @@ def within(self, timeframe: "Timeframe") -> models.QuerySet:
         )
 
 
-class ValueSetLookupQuerySet(CommittableQuerySet, ValueSetLookupQuerySetMixin):
+class CommittableQuerySet(BaseQuerySet, CommittableQuerySetMixin):
+    """A queryset for committable objects."""
+
+    pass
+
+
+class ValueSetLookupQuerySet(BaseQuerySet, ValueSetLookupQuerySetMixin):
```

Comment:
This class is fine, but I'm leaving a comment here because I can't leave it where I want to on line 180.

The superclasses of `ValueSetLookupByNameQuerySet` look incorrect. Can we fix that?

I think it should be this:

```python
class ValueSetLookupByNameQuerySet(BaseQuerySet, ValueSetLookupByNameQuerySetMixin):
```

Reply:
done